src/python/pants/base/exception_sink.py

```diff
@@ -174,15 +175,36 @@ def reset_log_location(cls, new_log_location):
     cls._pid_specific_error_fileobj = pid_specific_error_stream
     cls._shared_error_fileobj = shared_error_stream
 
+  class AccessGlobalExiterMixin:
```

Comment:
So, this mixin exists because of an aesthetic choice, of classes just being able to call `self._exiter` instead of the `ExceptionSink.global_exiter()`.

However, I agree that `get_global_exiter` has no reason to be private.

Reply:
Definitely open to feedback on the existence of the mixin. I like it, but not everybody might.